conf/swisnap-init.sh

```diff
@@ -16,6 +16,7 @@ swisnap_config_setup() {
     fi
 
     yq d -i ${CONFIG_FILE} log_path
+    yq w -i ${CONFIG_FILE} restapi.addr 0.0.0.0
```

Comment:
@costap this line fixed the failing probe checks

Reply:
Amazing stuff! Awesome